src/SOS/Strike/xplat/dbgeng.h

```diff
@@ -37,6 +37,12 @@ class DebugClient
         m_lldbservices2->AddRef();
     }
 
+    ~DebugClient()
+    {
+        m_lldbservices->Release();
```

Comment:
are m_lldbservices and m_lldbservices2 guaranteed to be non-null?

Reply:
Yes. Look at the constructor above. It also assumes they are never null. One of the reasons I move it to destructor.